dev-docs/bidders/mediaConsortium.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: bidder
+title: Media Consortium
+description: Prebid Media Consortium Bidder Adapter
+biddercode: mediaConsortium
+media_types: banner, video
+pbjs: true
+userId: all
+multiformat_supported: will-bid-on-any
+sidebarType: 1
+---
+
+### Prebid JS
```

Comment:
Please move the bid adapter parameters above this example and re-title the heading "Prebid JS configuration"

Reply:
If your adapter doesn't have bidder-specific parameters, please add that note.

There are 1 or 2 other adapters that don't take any kind of inventory parameter (e.g. "placement") but it can confuse publishers when it's missing if there's not a note saying "mediaConsortium doesn't require inventory parameters - we get it from the page URL" (or something along those lines)

Reply:
I've updated the documentation and prefaced it with something along the lines of what you suggested